ui/lib/date.ts

```diff
@@ -0,0 +1,5 @@
+
+export const dateToReadable = (date: Date | undefined): string | undefined => {
```

Comment:
Could you add a suite of unit tests (`date.test.ts`) for for this function?

Reply:
This is just a fn that was floating around freely that I moved to a separate file. I'd suggest we do this as a separate task

Reply:
@southbears Okay, I thought you added the function, but understand now that you were doing refactoring.  For the future, please do refactoring in separate pull requests so that refactoring and functionality don't get mixed together.